index.js

```diff
@@ -23,7 +25,7 @@ const main = async () => {
 		});
 		const repositories = getRepos.slice(0, getRepos.length - 1).split('\n');
 		const repoOSSF = {};
-		repositories.forEach((repository) => {
+		repositories.forEach(async (repository) => {
```

Comment:
`forEach` is sync, so it won't respect async functions.

to have serial tasks, we'd need:
```suggestion
		repositories.reduce(async (prev, repository) => {
		  await prev;
```